x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/prompts.ts

```diff
@@ -6,69 +6,69 @@
-  [
-    'system',
-    'You are a helpful assistant. ALWAYS use the provided tools. Use tools as often as possible, as they have access to the latest data and syntax. Always return value from ESQLKnowledgeBaseTool as is. Never return <thinking> tags in the response, but make sure to include <result> tags content in the response. Do not reflect on the quality of the returned search results in your response.',
-  ],
-  ['placeholder', '{chat_history}'],
-  ['human', '{input}'],
-  ['placeholder', '{agent_scratchpad}'],
-]);
+export const systemPrompts = {
+  openai: DEFAULT_SYSTEM_PROMPT,
+  bedrock: `${DEFAULT_SYSTEM_PROMPT} Use tools as often as possible, as they have access to the latest data and syntax. Always return value from ESQLKnowledgeBaseTool as is. Never return <thinking> tags in the response, but make sure to include <result> tags content in the response. Do not reflect on the quality of the returned search results in your response.`,
+  gemini:
+    `${DEFAULT_SYSTEM_PROMPT} ALWAYS use the provided tools, as they have access to the latest data and syntax.` +
+    "The final response is the only output the user sees and should be a complete answer to the user's question. Do not leave out important tool output. The final response should never be empty. Don't forget to use tools.",
+  structuredChat:
+    'Respond to the human as helpfully and accurately as possible. You have access to the following tools:\n\n' +
+    '{tools}\n\n' +
+    `The tool action_input should ALWAYS follow the tool JSON schema args.\n\n` +
+    'Valid "action" values: "Final Answer" or {tool_names}\n\n' +
+    'Use a json blob to specify a tool by providing an action key (tool name) and an action_input key (tool input strictly adhering to the tool JSON schema args).\n\n' +
+    'Provide only ONE action per $JSON_BLOB, as shown:\n\n' +
+    '```\n\n' +
+    '{{\n\n' +
+    '  "action": $TOOL_NAME,\n\n' +
+    '  "action_input": $TOOL_INPUT\n\n' +
+    '}}\n\n' +
+    '```\n\n' +
+    'Follow this format:\n\n' +
+    'Question: input question to answer\n\n' +
+    'Thought: consider previous and subsequent steps\n\n' +
+    'Action:\n\n' +
+    '```\n\n' +
+    '$JSON_BLOB\n\n' +
+    '```\n\n' +
+    'Observation: action result\n\n' +
+    '... (repeat Thought/Action/Observation N times)\n\n' +
+    'Thought: I know what to respond\n\n' +
+    'Action:\n\n' +
+    '```\n\n' +
+    '{{\n\n' +
+    '  "action": "Final Answer",\n\n' +
+    // important, no new line here
+    '  "action_input": "Final response to human"' +
+    '}}\n\n' +
+    'Begin! Reminder to ALWAYS respond with a valid json blob of a single action with no additional output. When using tools, ALWAYS input the expected JSON schema args. Your answer will be parsed as JSON, so never use double quotes within the output and instead use backticks. Single quotes may be used, such as apostrophes. Response format is Action:```$JSON_BLOB```then Observation',
+};
```

Comment:
The string concatenation (instead of template literals) in `structuredChat` may be obscuring what appears to be a missing closing backticks in the last set of instructions, and (nit) possibly some unintentional spacing (that should have no effect on the LLM), as illustrated by the following annotated diff:

![concat_vs_template_literal](https://github.com/user-attachments/assets/0047c1c2-d54e-44ca-b109-fdce25c08095)

Consider adopting the refactored `structuredChat` (as a template literal) below. This version passes a string `===` check with the original concatenated version. (It does not add additional backticks, or update the formatting.)

Consider updating it if the backticks are missing:

```typescript
export const systemPrompts = {
  openai: DEFAULT_SYSTEM_PROMPT,
  bedrock: `${DEFAULT_SYSTEM_PROMPT} Use tools as often as possible, as they have access to the latest data and syntax. Always return value from ESQLKnowledgeBaseTool as is. Never return <thinking> tags in the response, but make sure to include <result> tags content in the response. Do not reflect on the quality of the returned search results in your response.`,
  gemini: `${DEFAULT_SYSTEM_PROMPT} ALWAYS use the provided tools, as they have access to the latest data and syntax. The final response is the only output the user sees and should be a complete answer to the user's question. Do not leave out important tool output. The final response should never be empty. Don't forget to use tools.`,
  structuredChat: `Respond to the human as helpfully and accurately as possible. You have access to the following tools:

{tools}

The tool action_input should ALWAYS follow the tool JSON schema args.

Valid "action" values: "Final Answer" or {tool_names}

Use a json blob to specify a tool by providing an action key (tool name) and an action_input key (tool input strictly adhering to the tool JSON schema args).

Provide only ONE action per $JSON_BLOB, as shown:

\`\`\`

{{

  "action": $TOOL_NAME,

  "action_input": $TOOL_INPUT

}}

\`\`\`

Follow this format:

Question: input question to answer

Thought: consider previous and subsequent steps

Action:

\`\`\`

$JSON_BLOB

\`\`\`

Observation: action result

... (repeat Thought/Action/Observation N times)

Thought: I know what to respond

Action:

\`\`\`

{{

  "action": "Final Answer",

  "action_input": "Final response to human"}}

Begin! Reminder to ALWAYS respond with a valid json blob of a single action with no additional output. When using tools, ALWAYS input the expected JSON schema args. Your answer will be parsed as JSON, so never use double quotes within the output and instead use backticks. Single quotes may be used, such as apostrophes. Response format is Action:\`\`\`$JSON_BLOB\`\`\`then Observation`,
};
```

Reply:
Awesome! 

I'm also translating the `DEFAULT_SYSTEM_PROMPT` but not the rest. I meant to put this all in i18n, so I will do that is well if you think we should? The other option would be to have this all in English and instruct that the final response be in `responseLanguage`. That is what we do for the title prompt. However, I think sending a conversation in one language would get better results... what do you think?

Reply:
The `default` `i18n` translation and specific translations of the same prompt _may_ return slightly different results when sent to the same model.

Reply:
But what if the user's question is in a different language? Perhaps I should do some testing around this with my limited Español.

Reply:
Done! https://github.com/elastic/kibana/pull/191847/commits/f677a2280a5a839e67643477230eab731d9c6b52